cmd/operator/main.go

```diff
@@ -98,6 +100,8 @@ func main() {
 	flag.StringVar(&healthProbeAddr, "health-probe-bind-address", ":8081", "The address the health probe endpoint binds to.")
 	flag.Var(images, "images", fmt.Sprintf("Full images refs to use for containers managed by the operator. E.g thanos=quay.io/thanos/thanos:v0.33.0. Images used are %v", imagesUsed()))
 	flag.BoolVar(&openShiftEnabled, "openshift.enabled", false, "Enable OpenShift specific features such as Console Plugins.")
+	flag.StringVar(&otelCSVName, "opentelemetry-csv", "", "OpenTelemetry Operator starting CSV name. This can be used to install a specific OpenTelemetry Operator version. Empty string means the latest version will be installed.")
+	flag.StringVar(&tempoCSVName, "tempo-csv", "", "Tempo Operator starting CSV name. This can be used to install a specific Tempo Operator version. Empty string means the latest version will be installed.")
```

Comment:
> Empty string means the latest version will be installed.
Though this is not true when the CR specifies to not installs right?

Also keep in mind the args are set in the COO CSV shipped in the bundle. So this is not user configurable but rather to be set by the release team. Is that intended?
We did discuss passing the operant CSV name in the ClusterObs CR. This would allow to reference existing installs as well es install a new one.

Reply:
The question was rather how I can pass which CSV gets used. Right now its either "install the CSV yourself" or (iiuc) it'll use the first CSV with the right prefix in the operator namespace. I wonder if anyone will actually use `install: false`?

Reply:
> The question was rather how I can pass which CSV gets used. Right now its either "install the CSV yourself" or (iiuc) it'll use the first CSV with the right prefix in the operator namespace

This will/might be capability-specific. The tracing capability supports only rolling release so only head of the stable channel is supported. Once we support other capabilities, we can make changes to the spec, depending on the requirements.

>  I wonder if anyone will actually use install: false

This is not supported. If capability is enabled it will force to install the operators if they are not already installed. It is also clarified in the CR docs. The `install` option is there to support the use-case to instll only operators when the capability is disabled (e.g. for the MCOA use-case).

Reply:
Hm but the `install: false` case is documented in https://github.com/rhobs/observability-operator/blob/adeb53be7ec2f057ead8282d8af818152ffdefb5/docs/clusterobservability.md#deploy-capability-but-dont-deploy-the-operators.

But even the `install: true` case seems to be just another way to `kubectl create subscription.yaml` iiuc? Just now it needs two steps (install the operator, create the CR)?

Speaking of documentation: It references logging? https://github.com/rhobs/observability-operator/blob/adeb53be7ec2f057ead8282d8af818152ffdefb5/docs/clusterobservability.md#clusterobservability-crd
That isn't part of this feature anymore is it?